### PR TITLE
fix(@desktop/wallet): properly handle collectible groups in send modal

### LINKED ISSUE #14080: Wallet -> Send: do not show community NFTs in send modal
### Description

I am not really sure if we would support sending community NFTs through send modal ( currently only Assets are supported). Until then, lets hide the community minted NFTs in Send modal collectibles tab

Also, it looks like community minted NFT is treated as collection which is wrong

Screenshot 2024-03-21 at 16 46 34

### PR DESCRIPTION
**Pull request by dlipicar**
Fixes #14080

### What does the PR do

Community collectibles were not properly handled in the Send modal, showing a group of 0 elements instead and not allowing to select them for sending. This PR fixes that. 

There was a small rework in preparation for future tasks. Some other minor issues are also fixed and support for ERC1155 balance display is introduced.

https://github.com/status-im/status-desktop/assets/11161531/98ae94a4-8336-4bf0-8454-2756ddaeefd4

### CODE REVIEW

**Review by stefandunca — APPROVED** [review]
Great step towards having an integrated model for groups. However, I don't see order taken into consideration. Adding order support will be a great step towards simplifying the current approach.

**Comment by dlipicar** [comment]
> Great step towards having an integrated model for groups. However, I don't see order taken into consideration. Adding order support will be a great step towards simplifying the current approach.

@stefandunca yeah, this requires a proper refactor. When I did this there weren't any fancy utils to manage models in QML, we can probably use that to build a more generic solution now.

File: ui/imports/shared/stores/send/TransactionStore.qml [hunk 1]
```diff
@@ -157,14 +157,24 @@ QtObject {
...
             name: collectible.name,
             iconUrl: collectible.imageUrl,
-            collectionUid: collectible.collectionUid,
-            collectionName: collectible.collectionName,
-            isCollection: false
+            groupId: groupId,
+            groupName: groupName,
+            tokenType: collectible.tokenType,
+            itemType: itemType,
+            count: 1 // TODO: Properly handle count
```
Comment (Khushboo-dev-cpp) [inline ui/imports/shared/stores/send/TransactionStore.qml:177]: can we create a follow up task for this TODO please? or link to an existing github issue?

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM

File: ui/imports/shared/stores/send/TransactionStore.qml (hunk 1 above)
Comment (dlipicar) [reply ui/imports/shared/stores/send/TransactionStore.qml:177]: Here https://github.com/status-im/status-desktop/issues/14251

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/stores/send/TransactionStore.qml:177]: I meant if the link to that issue can be added to the code :)

**Pull request merged**